### DISCUSSION TITLE
The function of this tool is similar to Oracle TFA?

### DISCUSSION THREAD

**Discussion by plmqwsa** [oceanbase/obdiag#15] [Q&A]
The function of this tool is similar to Oracle Trace File Analyzer (TFA) ?

TFA link :https://oracle-base.com/articles/misc/trace-file-analyzer-tfa

**Comment by Teingi** [comment]
Compared to Oracle TFA, there are some features that have some similarities, but OceanBase Diagnostic Tool (obdiag) is a tool for diagnosing and troubleshooting issues in the OceanBase distributed database system. It provides a wide range of capabilities including log analysis, sql performance monitoring, system health checks and so on.
